src/dbnode/generated/thrift/rpc.thrift

```diff
@@ -490,6 +492,20 @@ struct Query {
   7: optional FieldQuery       field
 }
 
+struct AggregateTilesRequest {
+	1: required string sourceNameSpace
+	2: required string targetNameSpace
+	3: required i64 rangeStart
+	4: required i64 rangeEnd
+	5: required string step
+	6: bool removeResets
+	7: optional TimeType rangeType = TimeType.UNIX_SECONDS
+}
+
+struct AggregateTilesResult {
+    1: required i64 processedBlockCount
```

Comment:
Here too: Do you want to format everything to use tabs instead of spaces in this file? Looks like we are mixing them.

Reply:
Will do.